RecoBTag/Combined/python/DeepCSVJetTags_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+DeepCSVJetTags = cms.EDProducer(
+	'DeepFlavourJetTagsProducer',
+	src = cms.InputTag('DeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
+	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json'),
+  toAdd = cms.PSet(
+      probcc = cms.string('probc')
+      ),
+	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(DeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
+phase1Pixel.toModify(DeepCSVJetTags, checkSVForDefaults = cms.bool(True))
+phase1Pixel.toModify(DeepCSVJetTags, meanPadding = cms.bool(True))
+phase1Pixel.toModify(DeepCSVJetTags, toAdd = cms.PSet())
+
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(DeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
```

Comment:
what is the reason to maintain this era-based modification structure for the completely new (and not used in standard reco) taggers?
Please use RecoBTag/Combined/data/DeepCSV_PhaseI.json for everything.
Past discussion was suggestive that it performs just about as well for phase-0.
Also, the concerns about applicability to phase-0 are even smaller considering that this is meant for HLT developments which should become useful only for 2017.